docs/05-iam/domain-manager-setup-and-usage.md

```diff
@@ -0,0 +1,233 @@
+# Domain Manager setup and usage
+
+> **NOTE**
+> The following documentation refers to a SCS standard that is still in draft state.
+> It is not meant for productive use yet but CSPs are encouraged to test-drive and provide feedback!
+
+## Preface
+
+SCS defines the **Domain Manager** standard, introducing a special role to the OpenStack Keystone identity manager.
+This role offers a properly domain-scoped permission set to manage users, groups, projects and role assignments within a domain.
+Its intended use case is to offer extensive identity management self-service capabilities to tenants mapped to a domain.
+
+This guide will explain setup, configuration and usage of the SCS Domain Manager standard.
+
+### Warning regarding the exposure of domain names
+
+Due to architectural limitations currently existing in OpenStack Keystone, assigning the Domain Manager role to users will **enable them to see the IDs and names of all existing domains**.
+This includes domains other than their own, meaning that other tenant's identities might be exposed depending on the relation between them and the name of their domain.
+CSPs aiming to appoint Domain Manager users must be aware of this limitation and should exclusively **use pseudonymized domain names across the whole infrastructure**.
```

Comment:
Maybe a link to the standard for naming convention can help here. 
https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0301-v1-naming-conventions.md

Reply:
Good point! I've added this to the text. Thank you!